### ISSUE TITLE
Increase socket timeout

### ISSUE THREAD

**Issue by komandante** [bsorrentino/maven-confluence-plugin#241]
Our Confluence is very slow and sometimes we get error "error generating report: java.net.SocketTimeoutException: timeout: Socket closed"
It would be great adding connect, read and write timeouts to plugin parameters.

**Comment by bsorrentino** [comment]
Hi @komandante thanks for feedback this is a very useful feature

**Comment by bsorrentino** [comment]
Hi @komandante 

I've done the fix and deployed it in `7.0-SNAPSHOT` release.

Please, take a chance to test it and let me know if it really fix your issue 

The new timeout configuration is:

```xml
<configuration>

  <connectTimeoutInSeconds>10</connectTimeoutInSeconds>
  <readTimeoutInSeconds>10</readTimeoutInSeconds>
  <writeTimeoutInSeconds>10</writeTimeoutInSeconds>

<configuration>
```

**Comment by bsorrentino** [comment]
ix released in `7.0-beta2`
